templates/supplier_app/invoice-list.html

```diff
@@ -46,7 +46,7 @@ <h1 class="text-center"> {% trans "My Invoices"%}</h1>
                 {% endif %}
                 {% for invoice in object_list %}
                 <tr>
-                    <th scope="row">{{invoice.id}}</th>
+                    <th scope="row">{{ forloop.counter }}</th>
                     <td>{{invoice.invoice_date_received}}</td>
                     <td>{{invoice.invoice_number}}</td>
                     <td>{{invoice.po_number}}</td>
```

Comment:
Una opción para no ser tan repetitivo en estas partes del template es usar for loops, pero eso requeriría que tengas los valores en alguna variable iterable

Reply:
Podría usar una constante que tenga todos los fields que quero mostrar? no se que tan conveniente sea eso